base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 400px;
+  height: 75px;
+  transition-duration: 5s;
+}
+
+.navigationBarToggle {
+  display: flex;
+  flex-direction: row;
```

Comment:
This is the default, you don't need it

Reply:
Thx, same like the case before.

flex direction row (removed since default)